app.py

```diff
@@ -0,0 +1,9 @@
+from flask import Flask, render_template
+app = Flask(__name__)
+
+@app.route("/")
+def home():
+    return render_template('home.html')
+
+if __name__ == '__main__':
+    app.run(debug=True)
```

Comment:
Might want to consider specifying a host and a port:
`app.run(debug=True, host='0.0.0.0', port=3000)`